test/scanner/Tests/ScanLetSpec.hs

```diff
@@ -95,32 +137,121 @@ spec = do
     scanTokenTypes input `shouldBe` Right output
 
   it "supports explicit braces around definition" $ do
-    pendingWith "help needed"
-    -- let a = { a_block_1_1
-    --  a_block_1_2 ;
-    --      a_block_2_1
-    --        a_block_2_2 }
+    let input = unlines
+          [ "let a = { a_block_1_1"
+          , "            a_block_1_2;"
+          , "          a_block_2_1"
+          , "            a_block_2_2 }"
+          , "outside_let"
+          ]
+        output =
+          [ TLet
+          , TLbrace
+          , TId "a"
+          , TEq
+          , TLbrace
+          , TLbrace -- Inserts extra brace pair?
+          , TId "a_block_1_1"
+          , TId "a_block_1_2"
+          , TSemicolon
+          , TId "a_block_2_1"
+          , TId "a_block_2_2"
+          , TRbrace
+          , TRbrace
+          , TRbrace
+          , TSemicolon
+          , TId "outside_let"
+          ]
+    scanTokenTypes input `shouldBe` Right output
 
   it "supports explicit braces around bind group" $ do
     pendingWith "help needed"
     -- let { a = b
     --   and b = a }
     --
     -- though this looks rather unsightly
+    -- TODO(hans): The scanner isn't actually OK with this
```

Comment:
I don't think the scanner is looking for the `and` token, that might be the problem.